src/pages/pt-br/guides/fonts.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Usando fontes personalizadas
+description: Buscando adicionar algumas tipografias personalizadas a um site Astro? Use o Google Fonts com Fontsource ou adicione uma fonte de sua escolha. 
+i18nReady: true
+layout: ~/layouts/MainLayout.astro
+setup: |
+    import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro';
+---
+
+O Astro suporta todas as estratégias mais comuns para adicionar fontes personalizadas ao projeto de seu site. Este guia lhe mostrará duas opções diferentes para incluir fontes da web em seu projeto.
+
+## Usando um arquivo de fonte local
+
+Se você quiser adicionar arquivos de fontes ao seu projeto, recomendamos adicioná-los ao seu [`public/` directory](/en/core-concepts/project-structure/#public). Em seu CSS você pode então registrar as fontes com uma declaração[`@font-face` ]([https://developer.mozilla.org/pt-BR/docs/Web/CSS/@font-face]) e usar a propriedade `font-family` para estilizar seu site.
```

Comment:
Couple of broken links here:

- internal links need to start with the current language’s tag, `/pt-br/` in this case
- looks like the `@font-face` URL got wrapped in `[]` by accident!

```suggestion
Se você quiser adicionar arquivos de fontes ao seu projeto, recomendamos adicioná-los ao seu [`public/` directory](/pt-br/core-concepts/project-structure/#public). Em seu CSS você pode então registrar as fontes com uma declaração[`@font-face` ](https://developer.mozilla.org/pt-BR/docs/Web/CSS/@font-face) e usar a propriedade `font-family` para estilizar seu site.
```